btservice/btservice.cpp

```diff
@@ -37,7 +37,35 @@ btservice::btservice(openauto::configuration::IConfiguration::Pointer config)
     {
         OPENAUTO_LOG(info) << "[btservice] Service registered, port: " << cServicePortNumber;
     }
+    connectToBluetooth(QBluetoothAddress(QString::fromStdString(config->getLastBluetoothPair())), address);
 }
 
+void btservice::connectToBluetooth(QBluetoothAddress addr, QBluetoothAddress controller)
+{
+    // The raspberry pi has a really tough time using bluetoothctl (or really anything) to connect to an Android phone
+    // even though phone connecting to the pi is fine.
+    // I found a workaround where you can make the pi attempt an rfcomm connection to the phone, and it connects immediately
+    // This might require setting u+s on rfcomm though
+    // Other computers with more sane bluetooth shouldn't have an issue using bluetoothctl
+    
+    #ifdef RPI
```

Comment:
can you follow the openauto stye for the ifdefs (which is (ugh) no indentation

Reply:
Yea I'll fix that 😔